includes/Core/Assets/Assets.php

```diff
@@ -206,17 +206,19 @@ public function enqueue_asset( $handle ) {
 	 * @since 1.0.0
 	 */
 	public function enqueue_fonts() {
-		if ( $this->fonts_enqueued ) {
-			return;
-		}
```

Comment:
Let's keep this condition to avoid unnecessarily invoking the filter when it isn't going to be output anyways. My last comment was just about moving the setting of the property until after the filter to avoid a situation where it could be set but no fonts were actually enqueued.

Reply:
👍 Condition restored at the beginning of the method.